### PR TITLE
rm: Add info on the "storage_watermark" param of aklite

### PR DESCRIPTION
**Pull request by mike-sul**
Signed-off-by: Mike Sul <mike.sul@foundries.io>

### CODE REVIEW

**Comment by doanac** [comment]
Docs for 61940b388a7cf2c4edb1dcaf5d97a886aa25d01a are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1362/docs/artifacts/html/index.html

**Review by kprosise — APPROVED** [review]
One suggestion, otherwise it LGTM

**Review by doanac — APPROVED** [review]

File: source/reference-manual/ota/aktualizr-lite.rst [hunk 1]
```diff
@@ -109,6 +109,12 @@ The following are aktualizr-repo's configuration parameters that can be useful t
     # If the param is set to "0" then the App containers are (re-)created just after a successful boot on a new ostree version during aklite startup.
     create_containers_before_reboot = "0"
 
+    # A percentage of an available storage that can be used by Compose Apps.
+    # aktualizr-lite checks whether there is enough storage available before performing OTA update of Compose Apps.
+    # min(sizeof(AppsV_N+1) - sizeof(AppsV_N), 0)  <  <available_storage> * <storage_watermark>/100
+    # By default, if the configuration param is not specified, is set to "80".
```
[the remark this answers is not in the document]
Comment (mike-sul) [reply source/reference-manual/ota/aktualizr-lite.rst]: updated, thx.

**Comment by doanac** [comment]
Docs for 8b1ee8e1acba7ac7de63fd828250ffb290127c1e are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1363/docs/artifacts/html/index.html

**Pull request merged**